src/message/dht.d.ts

```diff
@@ -28,19 +28,34 @@ export class Record implements IRecord {
     constructor(p?: IRecord);
 
     /** Record key. */
-    public key: Uint8Array;
+    public key?: (Uint8Array|null);
```

Comment:
was this generated?

Reply:
Yes, I'm not sure why this changed, new patch release of protobufjs I guess?